RELEASES.md

```diff
@@ -0,0 +1,188 @@
+# Versioning and Releases
+
+This document details the versioning and release plan for Moby.
+This document is a living document, if you want to see a change to releases or
+versioning, propose the change by modifying this document.
+
+Since Moby is a critical piece of infrastructure software with a lot of
+history, one change that wil not be accepted is one that modifies the backwards
+compatability guarentees for existing releases.
+
+The core principal here being that we do not make breaking changes to a released
+API. New functionality can be added by bumping the API version, but not at the
+expense of changing the old behavior.
+
+
+## Versioning Scheme
+
+Moby generally follows semantic versioning with `<Major>.<Minor>.<Patch>`.
+
+The major version is bumped to signify a breaking change. In general this will
+not ever be bumped since we do not accept breaking API changes.
+
+Minor version is bumped when there is a release which includes new features.
+
+Patch version is bumped for bug-fix only releases.
+
+This repo includes a lot of history, including tags from Docker-CE releases.
+These will need to stay in place so as not to break people targetting these old
+tags. To find out latest release information see the [support](#support-horizon)
+section of this document.
+
+## Releases
+
+There is one release of Moby, `1.x`. New API's can be added, but must not break
+existing functionality.
+
+There will be two feature releases per year, roughly 6 months apart, the exact
+dates will be determined based on quality rather than timelines. The release
+cycle may be lengthened over time but generally should not be shortened.
```

Comment:
Why not shortened? Six months is a long time.

Reply:
I can remove the callout here. Mainly added because this is really focused around stability, and much shorter does not allow for very much time to stabilize.

Reply:
Removed.